### DISCUSSION TITLE
Using LNbank with CLN

### DISCUSSION THREAD

**Discussion by Sjors** [btcpayserver/btcpayserver#5596] [Ideas / Feature Requests]
My BTCPay server is connected to my own CLN node.

When I installed the LNBank plugin it complains:

```
LNbank requires an internal Lightning node to be configured.
```

Why?

**Comment by Kukks** [comment]
> My BTCPay server is connected to my own CLN node.
> 
> When I installed the LNBank plugin it complains:
> 
> ```
> LNbank requires an internal Lightning node to be configured.
> ```
> 
> Why?

Is it connected through a connection string on the store, or did you configure the internal node config on btcpay and use the "use internal node" option on stores?

**Reply by Sjors** [reply]
It was a connecting string before, changed it based on the comment below.

**Comment by dennisreimann** [comment]
@Sjors What kind of deployment method do you use? This might help: https://docs.btcpayserver.org/LNbank/Troubleshooting/#manual-deployment

**Reply by Sjors** [reply]
Faking (?) an internal node configuration did the trick!

**Reply by dennisreimann** [reply]
This isn't faking, but actually configuring it ;)

I assume you are using the manual deployment then. Basically the Docker deployment does exactly the same thing, but automatically.

**Reply by Sjors** [reply]
Yes it's manually deployed.
